plasmapy/formulary/collisions/helio/collisional_analysis.py

```diff
@@ -0,0 +1,360 @@
+
+    .. math::
+
+        T_{\rm i} = \frac{2T_{{\rm i}, \perp} + T_{{\rm i}, \parallel}}{3} \, ,
+
+    where :math:`T_{{\rm i}, \perp}` and :math:`T_{{\rm i}, \parallel}`
+    are the temperature of the :math:`{\rm i}`-particles along the
+    axes perpendicular and parallel to the ambient magnetic field.
+
+    In order to determine how extensively an individual parcel of
+    plasma has been processed by Coulomb collisions, :cite:t:`maruca:2013`
+    introduced an approached called collisional analysis. Which seeks
+    to quantify how collisions affect the plasma’s departures from
+    LTE, the equation for collisional thermalization
+    from :cite:t:`maruca:2013` is given below:
+
+     .. math::
+
+        \frac{d \theta_{21}}{dr} = A \left (
+        \frac{n_1}{v_1 T_1^{3/2}} \right ) \frac{\left( \mu_{1} \mu_{2}
+        \right )^{1/2} Z_{1} Z_{2} \left( 1 - \theta_{21} \right )
+        \left(1 + \eta_{21}\theta_{21} \right )}{\left(
+        \frac{\mu_{2}}{\mu_{1}} + \theta_{21} \right )^{3/2}} \lambda_{21}
+
+    and
+
+    .. math::
+
+         \lambda_{21} = 9 + \ln \left| B \left ( \frac{T^{3}_{1}}{n_{1}}
+         \right )^{1/2} \left( \frac{\theta_{21} +
+         \frac{\mu_{2}}{\mu_1}}{Z_{1}Z_{2}(\mu_{1} + \mu_{2})} \right )
+         \left( \frac{n_{2}Z_{2}^{2}}{n_{1}Z_{1}^{2}} + \theta_{21}
+         \right)^{1/2}\right |
+
+    With :math:`\eta = \frac{n_{2}}{n_{1}}`, :math:`\theta =
+    \frac{T_{2}}{T_{1}}`, :math:`A = 2.60 \times 10^{7} \, {\rm cm}^{3}
+    \, {\rm km} \, {\rm K}^{3/2} \, {\rm s}^{-1} \, {\rm au}^{-1}`,
+    and :math:`B = 1 \, {\rm cm}^{-3/2}{\rm K}^{-3/2}`.
+
+    The thermalization is from Coulomb collisions, which assumes
+    “soft”, small-angle deflections mediated by the electrostatic
+    force :cite:t:`baumjohann:1997`. It is assumed that there is no
+    relative drift between the ion species and that it is a mixed ion
+    collision, the Coulomb logarithm for a mixed ion collision is
+    given by :cite:t:`nrlformulary:2019`.
+
+    The density, velocity and temperature of the primary ion can be
+    radially scaled, as seen below. The values for the scaling can be
+    altered, though the default values are taken from
+    :cite:t:`hellinger:2011`.
+
+    .. math::
+
+        n(r) \propto r^{-1.8}\ , \hspace{1cm} v_{r}(r) \propto r^{-0.2}\ ,
+        \hspace{0.5cm} {\rm and} \hspace{0.5cm} T(r) \propto r^{-0.74}
+
+    Application is primarily for the solar wind.
+
+    Examples
+    --------
+    >>> import astropy.units as u
+    >>> from plasmapy.formulary.collisions import helio
+    >>> r_0 = [0.1, 0.1, 0.1] * u.au
+    >>> r_n = [1.0, 1.0, 1.0] * u.au
+    >>> n_1 = [1200, 1500, 1400] * u.cm**-3
+    >>> n_2 = [12, 18, 8] * u.cm**-3
+    >>> v_1 = [450, 350, 400] * u.km / u.s
+    >>> T_1 = [1.5 * 10**5, 2.1 * 10**5, 1.7 * 10**5] * u.K
+    >>> T_2 = [2.5 * 10**6, 1.8 * 10**6, 2.8 * 10**6] * u.K
+    >>> ions = [Particle("p+"), Particle("He-4++")]
```

Comment:
```suggestion
    >>> ions = ["p+", "He-4++"]
```
[optional] Would this make it more readable, since the conversion to a `ParticleList` will handle it?